minstall.cpp

```diff
@@ -2911,21 +2901,19 @@ bool MInstall::eventFilter(QObject* obj, QEvent* event)
 }
 
 // run before closing the app, do some cleanup
-void MInstall::cleanup()
+void MInstall::cleanup(bool endclean)
 {
     qDebug() << "+++" << __PRETTY_FUNCTION__ << "+++";
     if (pretend) return;
 
-    shell.run("command -v xfconf-query >/dev/null && su $(logname) -c 'xfconf-query --channel thunar-volman --property /automount-drives/enabled --set " + auto_mount.toUtf8() + "'");
-
-
-    shell.run("cp /var/log/minstall.log /mnt/antiX/var/log >/dev/null 2>&1");
-    shell.run("rm -rf /mnt/antiX/mnt/antiX >/dev/null 2>&1");
-    sync();
+    if (endclean) {
+        shell.run("command -v xfconf-query >/dev/null && su $(logname) -c 'xfconf-query --channel thunar-volman --property /automount-drives/enabled --set " + auto_mount.toUtf8() + "'");

```

Comment:
Won't we need to set automount to the default setting regardless if endclean is true or false?

Reply:
There is another such line which I think disables the automount, and this is done right when the setup is started.

When the install is aborted for whatever reason, endclean is false. If we set the automount back to defaults then we may be in trouble if the user tries to continue the setup again after aborting it.

If the user ends up closing the setup, cleanup() is run anyway and endclean is true so this code should always run at the end regardless of whether or not the setup is aborted.

Reply:
Thanks, makes sense.